src/components/enterprise-subsidy-requests/data/hooks.js

```diff
@@ -53,16 +54,21 @@ export function useSubsidyRequests(subsidyRequestConfiguration) {
   const fetchSubsidyRequests = async (subsidyType) => {
     setIsLoading(true);
     try {
+      const { email: userEmail } = getAuthenticatedUser();
+      const { enterpriseCustomerUuid: enterpriseUUID } = subsidyRequestConfiguration;
```

Comment:
[clarification] I think it might be possible for `subsidyRequestConfiguration` to be null here? If so, this object destructing will cause an error.

Reply:
oh here it's not possible since this function only gets called if subsidyRequestConfiguration?.subsidyRequestsEnabled === true